hibernate-agroal/src/main/java/org/hibernate/agroal/internal/AgroalConnectionProvider.java

```diff
@@ -79,7 +91,7 @@ private static <T> void copyProperty(Map<String, Object> properties, String key,
 
 	@Override
 	public void configure(Map<String, Object> props) throws HibernateException {
-		LOGGER.debug( "Configuring Agroal" );
+		ConnectionInfoLogger.INSTANCE.configureConnectionPool( AGROAL_CONFIG_PREFIX );
```

Comment:
There's some non-uniformity in the string passed to `configureConnectionPool`.

Instead of passing the config property prefix, I think I would just log `Agroal` or perhaps `AgroalConnectionProvider`.